modules/administration-guide/partials/proc_viewing-che-metrics-on-grafana-dashboards.adoc

```diff
@@ -1,27 +1,35 @@
 // monitoring-{prod-id-short}
 
 [id="viewing-{prod-id-short}-metrics-on-grafana-dashboards"]
-= Viewing {prod-short} Server metrics on Grafana dashboards
+= Viewing {prod-short} Server from an OpenShift web console dashboard
 
-To view the {prod-short} Server metrics on Grafana:
+After configuring the in-cluster Prometheus instance to collect {prod-short} Server JVM metrics, the metrics can be viewed from a custom dashboard within the *Administrator* perspective of the OpenShift web console.
 
 .Prerequisites
 
-* Prometheus is collecting metrics on the {prod-short} cluster. See xref:monitoring-with-prometheus-and-grafana.adoc[].
+* An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 
-* Grafana 6.0 or later is running on port `3000` with a corresponding Service. See link:https://grafana.com/docs/grafana/latest/installation/kubernetes/[Installing Grafana].
+* An instance of {prod-short} running in {orch-name}.
 
+* The in-cluster Prometheus instance is collecting metrics. See xref:collecting-che-metrics-with-prometheus[].
 
 .Procedure
 
-. Add the data source for the Prometheus instance. See link:https://prometheus.io/docs/visualization/grafana/#creating-a-prometheus-data-source[Creating a Prometheus data source].
+. Create a ConfigMap in the `openshift-config-managed` {orch-namespace} containing the dashboard definition and apply the necessary label. The dashboard definition is based on Grafana 6.x dashboards, however, not all Grafana 6.x dashboard features are supported within the OpenShift web console.
```

Comment:
I suggest moving and placing the second sentence after the code block because it contains no action steps. Also, is this what you meant?
```suggestion
. Create a ConfigMap for the dashboard definition in the `openshift-config-managed` {orch-namespace} and apply the necessary label.
```